protocol/test/ConvertUnripe.test.js

```diff
@@ -586,21 +586,21 @@ describe('Unripe Convert', function () {
       // CHECK TO SEE THAT RECAP AND PENALTY VALUES ARE UPDATED AFTER THE CONVERT
```

Comment:
Some math would be great here to show the correctness of the values.

Reply:
Penalty Maths
        params before convert: supply = 10000,000000 , s.u[unripeToken].balanceOfUnderlying = 1000,000000
        penalty before convert ---> (s.u[unripeToken].balanceOfUnderlying ** 2).mul(amount).div(supply ** 2) = 1e9 ^ 2 * 
          1e6 / 1e10 ^ 2 = 0.01
        // convert happens, 100e6 removed from unripe supply, 1e6 removed from underlying ripe
        // new params: supply = 9900,000000 , s.u[unripeToken].balanceOfUnderlying = 999,000000
        // penalty after convert ---> (s.u[unripeToken].balanceOfUnderlying ** 2).mul(amount).div(supply ** 2) = 999000000 ^ 2 * 1e6 / 9900000000 ^ 2 = 0.010182

Reply:
Penalty Maths
        -params before convert: supply = 10000,000000 , s.u[unripeToken].balanceOfUnderlying = 1000,000000
        - penalty before convert ---> (s.u[unripeToken].balanceOfUnderlying ** 2).mul(amount).div(supply ** 2) = 1e9 ^ 2 * 
         1e6 / 1e10 ^ 2 = 0.01
        - convert happens, 100e6 removed from unripe supply, 1e6 removed from underlying ripe
        - new params: supply = 9900,000000 , s.u[unripeToken].balanceOfUnderlying = 999,000000
        - penalty after convert ---> (s.u[unripeToken].balanceOfUnderlying ** 2).mul(amount).div(supply ** 2) = 999000000 ^ 2 * 1e6 / 9900000000 ^ 2 = 0.010182

Reply:
GetUnderlying Maths
 - unripe amount :  1000000
 -supply:  9900000000 (same as above)
 - s.u[unripeToken].balanceOfUnderlying:  999000000 (same as above)
 - s.u[unripeToken].balanceOfUnderlying.mul(unripe).div(supply) = 999000000 * 1000000 / 9900000000 = 100909